### DISCUSSION TITLE
group by blueprint

### DISCUSSION THREAD

**Discussion by paulhaem** [statamic/cms#4896] [Troubleshooting] [answered]
Hello,
when using `{{ entries group_by="blueprint" }}` on a collection it outputs `ErrorException: array_key_exists()`. However when going through the dump or using `group_by="id"` blueprint is available as a string at the item level.

**Comment by edalzell** [comment, marked as answer]
sounds like a bug I guess?

**Reply by paulhaem** [reply]
Thanks for the help! I created an issue for this: https://github.com/statamic/cms/issues/4898
